### DISCUSSION TITLE
formula in design

### DISCUSSION THREAD

**Discussion by nooyou** [Budibase/budibase#3966] [Help]
Hello community.
Great tool, thanks. I have a question that I hope is not too stupid. When I build my database in 'data' I have created a 'formula', but when I go to 'design' the formula does not appear. How can I see it?
Thanks for your help.

**Comment by mjashanks** [comment]
Hi @nooyou 

I am unable to reproduce your issue. My Steps
1. Create a table, with a formula column - I named my table "Invoices" and my column "Tax Amount"
2. In design section, I added a "Table Block", and selected my "Invoices" as a data source
The "Tax Amount" column is displayed in a table

I repeated this with a  
```
> Repeater Block : Data source = Invoices
    > Paragraph: Text = {{ Repeater Block.Invoices.Tax Amount }} (which I selected from the binding menu)
```

This worked also.

Where are you expecting the formula to appear?

**Comment by nooyou** [comment]
Yes, your example is good but what if I want to add a total column that updates when I change the vat column?

**Reply by mjashanks** [reply]
The column that I added was a Formula column.... I didn't mention the other columns that I have:

- Description (Text)
- Amount Ex Tax (Number)
- Tax Amount - Formula, in Javascript = `return $("Amount Ex Tax") *  0.2;` (i.e. 20% tax rate)

All of these columns display in my table, or are available for Binding. The "Tax Amount" column automatically changes when I update the "Amount Ex Tax" value.

**Comment by nooyou** [comment]
In 'design' 'Field Group' formula does not appear at my place.

**Reply by mjashanks** [reply]
You mean when you click "Update Form Fields" ?

If so, then this is because it's not an "Input" field - i.e. its readonly.

You can still display it though. Inside the Field Group, add a new "Paragraph" component. Then you can set it's `Text` property.. e.g. 

`Tax: ${{ New Repeater block.Invoices.Tax Amount }} `

...which displays...

`Tax: $20`

**Comment by nooyou** [comment]
Ok great thanks I'll look into it.
